forge/src/runner.rs

```diff
@@ -54,27 +49,33 @@ pub struct TestResult {
     ///
     /// If this is the result of a fuzz test (`TestKind::Fuzz`), then this is the median of all
     /// successful cases
+    // TODO: The gas usage is both in TestKind and here. We should dedupe.
```

Comment:
I'm not sure why we have a `gas_used` in `TestResult` as well as `TestKind` - leftover?